BuildCustomFunctions.ps1

```diff
@@ -405,23 +428,13 @@ $($IndentChar)$($IndentChar)Return `$Results"
                 Write-Host ("[STATUS] Building: $NewCommandName") -BackgroundColor:('Black') -ForegroundColor:('Magenta') # | Tee-Object -FilePath:($LogFilePath) -Append
                 $OutputFilePath = "$CustomFolderPath/$NewCommandName.ps1"
                 [System.IO.File]::WriteAllLines($OutputFilePath, $NewScript, (New-Object System.Text.UTF8Encoding $true))
-                # Validate script syntax
```

Comment:
Why remove this check?

Reply:
We bumped the PSScript Analyzer checks to the stage where we test the module. 

I thought it was strange to test the built functions for validity outside of the test module/ invoke pester steps which is where we've combined them for the PowerShell Module project.

Reply:
The intent was to validate that the `BuildCustomFunctions.ps1` created valid scripts. I guess as long as they are called in sequence it should be fine. Might make editing this script locally harder though.